docs/checks.md

```diff
@@ -588,3 +588,15 @@ possible.
 **Remediation steps**
 - Fix the vulnerabilities. The details of each vulnerability can be found on <https://osv.dev>.
 
+## Webhooks 
+
+Risk: `High`  (possible service can be accessed by third-party)
+
+This check validate if the webhook defined in the repository have a token configured to send as validation to make sure that is a trusted request.
+ 
+
+**Remediation steps**
```

Comment:
@olivekl can you advise on the text?

Reply:
Suggested rewording:
"Risk: `High`  (service possibly accessible to third parties)

This check determines whether the webhook defined in the repository has a token configured to authenticate the origins of requests."

Can we add something about how the check is scored?

Reply:
Let's add the the full score is given when all webhooks have a secret used for authentication. Otherwise, the score is calculated as a (the number of webhooks with secrets / total number of webhooks configured) * 10 .